public/scripts/theme.js

```diff
@@ -20,26 +20,26 @@
     }
 })();
 
-// eslint-disable-next-line
-function takePill(color) {
-    const html = document.getElementsByTagName('html')[0];
-    if (color === 'red') {
-        const storedTheme = localStorage.getItem('theme');
-        if (storedTheme === 'matrix') {
-            console.log("You've already entered the rabbit hole");
-        } else {
-            console.log('There is no turning back');
-            html.classList.add('theme-matrix');
-            html.classList.remove('theme-dark');
-            localStorage.setItem('theme', 'matrix');
-        }
-    } else if (color === 'blue') {
-        html.classList.remove('theme-matrix');
-        localStorage.removeItem('theme');
-    } else {
-        console.log(
-            'This is your last chance. After this there is no turning back. You take the blue pill, the story ends; you wake up in your bed and believe whatever you want to believe.',
-        );
-        console.log('You take the red pill, you stay in Wonderland and I show you how deep the rabbit hole goes');
-    }
-}
+// // eslint-disable-next-line
```

Comment:
should we delete this instead of commenting out?

Reply:
@ScaredIbis said to leave it in just in case we require it in the future

Reply:
sorry @TimJones97, I'm gonna go back and say just delete it. we can always copy paste from an older version of the repo if we want it back in the future